build-tools/api-xml-adjuster/api-xml-adjuster.targets

```diff
@@ -20,32 +21,52 @@
     </CreateItem>
   </Target>
 
-  <Target Name="_ClassParse"
-      BeforeTargets="_AdjustApiXml"
-      DependsOnTargets="_DefineApiFiles"
-      Inputs="%(ApiFileDefinition.ParameterDescription)"
-      Outputs="%(ApiFileDefinition.ApiAdjustedXml)">
+  <Target Name="_ClassParsePrepare"
+      DependsOnTargets="_DefineApiFiles">
     <PropertyGroup>
       <ClassParse>$(_TopDir)\bin\$(Configuration)\lib\xamarin.android\xbuild\Xamarin\Android\class-parse.exe</ClassParse>
     </PropertyGroup>
     <MakeDir Directories="$(_OutputPath)api" />
-    <Exec
+    <CreateItem
```

Comment:
CreateItem is deprecated and we should not be using it. You should use an `ItemGroup` instead

```
<ItemGroup>
    <_ClassParseCommands Include="@(ApiFileDefinition)">
        <Command>$(ClassParse)</Command>
       <Arguments>$(AndroidSdkDirectory)\platforms\android-%(ApiFileDefinition.Id)\android.jar -platform=%(ApiFileDefinition.Level) -parameter-names=&quot;%(ApiFileDefinition.ParameterDescription)&quot; -o=&quot;%(ApiFileDefinition.ClassParseXml)&quot;</Arguments>
   </_ClassParseCommands>
</ItemGroup>
```

The above *should work :)